pages/events/[id].vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+
+import Event from "~/components/events/Event.vue"
+
+const route = useRoute()
+let id: string;
+
+if (route.params.id == undefined) {
+  throw console.error("Failed to get param.")
+} else {
+  id = route.params.id as string
+}
+</script>
+
+<template>
+  <main>
+    <ClientOnly>
```

Comment:
you could avoid all the clientonly stuff by using `useFetch` 
that should take care of everything, even server side hydration

Reply:
but for now normal fetch is fine, just don't add more of it pls